### PR TITLE
update step 4 and 8

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4761

### CODE REVIEW

**Comment by e0i** [comment]
@mjcaparas The PR that addresses issue #4761 is available for review. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/powerbi-reports.md [hunk 1]
```diff
@@ -149,6 +149,9 @@ You can create a custom dashboard in Power BI Desktop to create visualizations t
 7. Click **File** > **Options and settings** > **Custom data connectors**.
 
 8. Select **New table and matrix visuals** and **Custom data connectors** and click **OK**.
+
+    >[!NOTE]
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/powerbi-reports.md]: Recommended MD compatibility rule: single space between quote marker and its subsequent text
```suggestion
    > [!NOTE]
```

File: windows/security/threat-protection/microsoft-defender-atp/powerbi-reports.md [hunk 2]
```diff
@@ -149,6 +149,9 @@ You can create a custom dashboard in Power BI Desktop to create visualizations t
...
+    >If you plan on using Custom Connectors or connectors that you or a third party have developed, you must select *(Not Recommended) Allow any extension to load without warning* under **Power BI Desktop** > **File** > **Options and settings** > **Options** > **Security** > **Data Extensions**".
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/powerbi-reports.md]: (same as above)
```suggestion
    > If you plan on using Custom Connectors or connectors that you or a third party have developed, you must select *(Not Recommended) Allow any extension to load without warning* under **Power BI Desktop** > **File** > **Options and settings** > **Options** > **Security** > **Data Extensions**".
```

**Comment by Dansimp** [comment]
added minor copy edit

**Pull request merged**